### ISSUE TITLE
Raw to RGB

### ISSUE THREAD

**Issue by ycjing** [kornia/kornia#652]
## 🚀 Feature

Hi. Thank you for the great work! Is there any plan or possible solution for raw-to-RGB (Bayer Pattern to RGB) transformation? The corresponding OpenCV function is as follows:
`img_bgr = cv2.cvtColor(img_bayer_bg.astype(np.uint8), cv2.COLOR_BAYER_BG2BGR)`
Thank you!

## Motivation

OpenCV raw-to-RGB function in a differentiable way.

## Pitch

A way to transform raw image to RGB image in a differentiable manner.

**Comment by edgarriba** [comment]
@ycjing this functionality it's definitely inside the scope of the library. We can add it to the roadmap for the upcoming releases

**Comment by ycjing** [comment]
Hi @edgarriba 

Thank you for the response! I appreciate it.

Best,
Yongcheng

**Comment by edgarriba** [comment]
@cceyda is this what you did with `rgb_to_linear` ?

**Comment by cceyda** [comment]
@edgarriba  No, rgb_to_linear reverses the gamma correction on an sRGB image.
Unfortunately I'm not familiar with Bayer Patterns, which looks to be a low level processing thing done by digital cameras 🤷 
I'm curious about your use case though @ycjing
